dev-util/visual-studio-code/visual-studio-code-1.22.2.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit check-reqs eutils xdg-utils
+
+MY_PN="vscode"
+MY_P="${MY_PN}"-"${PV}"
+
+DESCRIPTION="Microsoft's free and open-source code editor"
+HOMEPAGE="https://code.visualstudio.com/"
+SRC_URI="https://github.com/Microsoft/${MY_PN}/archive/${PV}.tar.gz
```

Comment:
`-> ${P}.tar.gz`

Reply:
This directly contradicts what mgorny said to do.